### PR TITLE
WIP: Update convertRaDecToCgemUnits class

### PR DESCRIPTION
**Pull request by steelewool**

### CODE REVIEW

**Comment by steelewool** [comment]
Resulting in conflicting files that will need to be resolved. Do you have time for that @zcsteele

**Comment by zcsteele** [comment]
@RobSteele49 Judging by the conflicts, it looks like git is trying to remove the serial I/O code during this merge.  I can't test the serial behavior from here; did you want to keep the serial interface code in `cgemController.py`?

**Comment by steelewool** [comment]
This is where the comment was. Yes I think it would be perfectly OK to pull out the serial i/f. It might make it easier to debug for both of us. I am planning another outing with the telescope tonight. Last night I never got the chance to observe - I just worked on the software.

File: cgemController.py [hunk 1]
```diff
@@ -50,39 +41,37 @@
...
+#            data = ser.read(1)
+#            print 'data : ', data
+#            if (data == '#'):
+#                print 'found the hash tag'
+#                foundHashTag = False
+#        ser.write(b'e')            # write a string
+#
+#        foundHashTag = True
+#        output = ''
+#        while (foundHashTag):
```
Comment (zcsteele) [inline cgemController.py]: @RobSteele49 I'm not clear on why this code block was duplicated - is this the pattern for getting an ACK from the telescope?  If so, it's a good candidate for a new `SerialInterface` object.

File: convertRaDecToCgemUnits.py [hunk 2]
```diff
@@ -32,14 +32,13 @@ def convertSeconds(self, seconds):
             self.strGotoValue += '0'
         return self.strGotoValue
 
-
-    def highMidLow(self, seconds):
```
Comment (zcsteele) [inline convertRaDecToCgemUnits.py]: @RobSteele49 as a sanity check, are we ready to remove this method from the interface?

File: convertRaDecToCgemUnits.py [hunk 3]
```diff
@@ -63,8 +62,12 @@ def toCgem(self):
         
         return self.convertSeconds(self.decInSeconds)
 
-if __name__ == '__main__':
+# This paradigm was provided by Zach as a way to test the individual
+# classes as a main program.
 
+if __name__ == '__main__':
+    
+    # Is there a better way to initialize the ra and dec values?
```
[the remark this answers is not in the document]
Comment (steelewool) [reply convertRaDecToCgemUnits.py]: sounds good.

**Comment by steelewool** [comment]
Let me know when you are ready for me to merge this. I've already merged my testSetup branch last night.

**Comment by zcsteele** [comment]
I'm ready to merge this whenever, I tested it & it looks OK to me.

**Pull request merged**